.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,29 @@
+name: E2E tests
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    container: docker.io/paritytech/ci-unified:bullseye-1.75.0-2024-01-22-v20240109
```

Comment:
is it possible to specify container only once in actions/run-polkadot ?

Reply:
I don't think it's possible to specify in actions/run-polkadot.
One way we could get rid of this duplicated setting in two workflows would be setting it in a [repository variable](https://github.com/paritytech/substrate-tip-bot/settings/variables/actions) - WDYT?

Reply:
yea, i think makes sense to update it from there once. Would be great to describe also in readme how to maintain the version of Release and Where to get according version container/image?

important because it makes sure to have similar rust and toolchain versions according to release version

Example https://github.com/paritytech/polkadot-sdk/blob/polkadot-v1.7.2/.github/workflows/fmt-check.yml#L18

Reply:
Ok, I introduced a repository variable and explained things in the readme.